subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/timeout/JavaProcessStackTracesMonitor.java

```diff
@@ -50,7 +50,7 @@
     private static final Pattern UNIX_PID_PATTERN = Pattern.compile("([0-9]+)");
 
     public static void main(String[] args) {
-        System.out.println(getAllStackTracesByJstack());
+//        System.out.println(getAllStackTracesByJstack());
```

Comment:
I think this doesn't quite do it.

We're printing out the stdout/stderr when running a command:
https://github.com/gradle/gradle/blob/1e1d28a4e9f6bb47406ab1272bc1bec74eb9a89f/subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/timeout/JavaProcessStackTracesMonitor.java#L169-L170

There are a couple of other places that we might expose stdout:
https://github.com/gradle/gradle/blob/1e1d28a4e9f6bb47406ab1272bc1bec74eb9a89f/subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/timeout/JavaProcessStackTracesMonitor.java#L191-L192

https://github.com/gradle/gradle/blob/1e1d28a4e9f6bb47406ab1272bc1bec74eb9a89f/subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/timeout/JavaProcessStackTracesMonitor.java#L197

Could we switch to using `jps -mm` instead of `ps x`? That would return only java processes.

Reply:
I printed the output to a file instead of stdout.